src/main/java/com/google/devtools/build/lib/bazel/rules/workspace/HttpFileRule.java

```diff
@@ -96,4 +96,7 @@ public Metadata getMetadata() {
 
  <p>Targets would specify <code>@my_deb//file</code> as a dependency to depend on this file.</p>
 
+ <p>You may also reference files on the localhost, using "file:///path/to/file" (on Unixes) or
```

Comment:
On first read, I thought the triple slashes were just for Windows, and got a little confused because it also shows up in the Unix example.

How does this sound?

"You may also reference files on the current system (localhost), by using `file:///path/to/file` if you're on Unix-based systems. If you're on Windows, use `file:///c:/path/to/file`. In both examples, note the three slashes `/` - the first two slashes belong to `file://`, and the last belongs to the absolute path to the file."

Reply:
Nice, thanks! Done.